frontend/src/components/object/ObjectUpload.vue

```diff
@@ -62,7 +62,7 @@ const onUpload = async (event: any) => {
     // Update object store
     await objectStore.fetchObjects({ bucketId: bucketId, userId: getUserId.value, bucketPerms: true });
   } else {
-    toast.add({ severity: 'error', summary: 'Error', detail: 'Failed to acquire bucket ID', life: 3000 });
+    toast.error('Uploading file', 'Failed to acquire bucket ID');
```

Comment:
Style: Technically this is an updateObject operation, so this should be something along the lines of `Updating file` at minimum. Consult with Tyler.